docs/source/usage/01_usage.md

```diff
@@ -134,6 +143,14 @@ $ ansible-playbook deploy-edpm.yml -K --tags bootstrap,packages [-e @scenarios/c
 $ ansible-playbook deploy-edpm.yml -K --skip-tags bootstrap,packages [-e @scenarios/centos-9/some-environment -e <...>]
 ```
 
+If you've already deployed OpenStack but it failed
```

Comment:
I'd push that new block just bellow the next one actually - else, the talk about "running twice with --tags and --skip-tags" doesn't make sense anymore at that point. Care to move it?

Reply:
I don't understand.

Block = lines 146 to 153 ?

If so, you want it at the "next one". Which lines is the next one?

"the talk about "running twice with --tags and --skip-tags" doesn't make sense anymore"

My example only uses --tags, it doesn't use --skip-tags. There's an example prior with --skip-tags but they are independent.

Move it where?

Reply:
I think I understand. I was parsing this as:

- example 1
- example 2
- general statement

but I think you were thinking that the "general statement" was a statement about example 1. With that in mind I believe I have made the requested change.

Reply:
thank you!